dapps/marketplace/src/css/app.css

```diff
@@ -34,10 +34,8 @@ body {
   height: 3rem;
 }
 
-/* sticky footer sibling */
 main {
   flex: 1 0 auto;
-  padding-bottom: 4rem;
```

Comment:
I think we still need this for desktop. For example the 'create listing' page on desktop has no margin at the bottom

Reply:
I removed it since it added some unnecessary padding on mobile. Didn't see that it affected Desktop, I can reset the padding on mobile with `@media` query then.

Thanks for notifying this.